docs/syntax_and_semantics/platform_support.md

```diff
@@ -55,7 +55,7 @@ Most typically, some parts of the standard library are not supported completely.
 | Target | Description | Supported versions | Comment |
 | ------ | ----------- | ------------------ | ------- |
 | `x86_64-windows-msvc` | x64 Windows (MSVC) | 7+ | :material-circle-slice-7: tests<br> :material-checkbox-marked-circle: builds |
-| `x86_64-windows-gnu` | x64 Windows (MinGW-w64) | 7+, MSYS2 `UCRT64` or `CLANG64` environment | :material-circle-slice-7: tests<br> :material-checkbox-marked-circle: builds |
+| `x86_64-windows-gnu` | x64 Windows (MinGW-w64) | 7+, MSYS2 `UCRT64` / `MINGW64` / `CLANG64` environment | :material-circle-slice-7: tests<br> :material-checkbox-marked-circle: builds |
 | `aarch64-windows-msvc` | ARM64 Windows (MSVC) | 11+ | :material-selection-ellipse: tests<br> :material-selection-ellipse: builds |
 | `aarch64-windows-gnu` | ARM64 Windows (MinGW-w64) | 11+, MSYS2 `CLANGARM64` environment | :material-selection-ellipse: tests<br> :material-selection-ellipse: builds |
```

Comment:
@HertzDevil Is `MINGW64` supported on `aarch64-windows-gnu`?

Reply:
You can run all the x86-64 MSYS2 environments on an ARM64 Windows machine as the OS provides the emulation, but they would still have the `x86_64-windows-gnu` triple, so it is not quite the same thing